packages/inter-protocol/src/stakeFactory/stakeFactoryManager.js

```diff
@@ -204,11 +204,7 @@ const manager = {
       brands.Attestation,
       X`Invalid Attestation ${attestationGiven}. Expected brand ${brands.Attestation}`,
     );
-    fit(
-      attestationGiven.value,
-      M.bagOf([M.string(), M.bigint()]),
-      'attestationGiven',
-    );
+    fit(attestationGiven.value, M.bagOf(M.string()), 'attestationGiven');
```

Comment:
It is a set of `[T, bigint]` pairs representing a multiset of `T` elements. Equally valid to describe as a single-valued map from `T` to bigint representing a multiset of `T` elements.

Taking "bag" to be synonymous with "multiset", it is a "bag of T".

However, I can see the "entries" terminology is confusing for producing the pairs. It only makes sense in terms of either view of the underlying representation. Nevertheless, it is the right API to visit the bag in O(unique elements) rather than O(elements).

What change do you think is called for?

Reply:
Actually, there's an API change that works as well with the bag/multiset view, and works even better with the entries/single-valued-mapping view.

```js
M.bagOf(elementPatt?: Pattern, countPatt?: Pattern)
```
which is more consistent with the existing
```js
M.mapOf(keyPatt?: Pattern, valuePatt?: Pattern)
```

Although it is hard to imagine a scenario where the countPatt is useful, it is less surprising to allow it than to omit it. I will make this change.

Reply:
Done. PTAL

Reply:
> However, I can see the "entries" terminology is confusing for producing the pairs

Ah yes, that's what confused me.

> What change do you think is called for?

I'll think on it. Out of scope here, I reckon.